docs/deprecated_runbooks/VirtualMachineCRCErrors.md

```diff
@@ -1,5 +1,7 @@
 # VirtualMachineCRCErrors
 
+<!-- This file is deprecated, see VMStorageClassWarning for the relevant and up-to-date runbook content -->
```

Comment:
I believe this comment is relevant also to upstream and should be visible

Reply:
Hi @jherrman, WDYT?

Reply:
Agreed, changed the line to be explicit.